gnd/src/main/java/com/google/android/gnd/model/feature/Feature.java

```diff
@@ -39,6 +39,10 @@ public boolean isGeoJson() {
     return getGeoJsonString() != null;
   }
 
+  public boolean isPolygon() {
```

Comment:
a polygon can be a subset of a GeoJson string. Do we need something different here ? Try it out here 
https://geojson.io/#map=6/15.475/75.839

Reply:
Re "GeoJson" vs "Polygon", it's worth mentioning that at the user/functional level, this feature is a specifically a polygon drawing tool, and that GeoJson just so happens to be how the polygons are serialized. When we added support for imported GeoJson, we took the "shortcut" of not deserializing the GeoJson into an application-specific data model, leading to a leaky abstraction here.

Since `Feature` is part of the application/domain model, I wouldn't be opposed to exposing polygon as a distinct attribute of feature, with an eye towards decoupling the GeoJson representation of the objects from the application model. In the future they should probably remain local and remote representations.